tests/foreman/api/test_http_proxy.py

```diff
@@ -263,3 +263,66 @@ def test_positive_assign_http_proxy_to_products():
         assert r.http_proxy_id == http_proxy_b.id
 
     product_a.sync({'async': True})
+
+
+@pytest.mark.tier2
+def test_positive_sync_proxy_with_certificate(request, target_sat, module_org, module_product):
+    """Assign http_proxy with ca.crt to Repo and test
+     whether http_proxy and cert are used during sync.
+
+       Steps:
+       - Generate cert files with custom_cert_generate,
+         collect cacert path.
+       - Create new org, product, and http-proxy with path to cacert.
+       - Create repo using selected http proxy, perform repo sync.
+
+    :expectedresults: http-proxy with cacert is assigned to repo,
+    sync operation uses assigned http-proxy with the cert.
+
+    :id: a9645b7f-228e-4f4d-ab04-610382bd2d0b
+
+    :BZ: 2144044
+
+    :customerscenario: true
+
+    :Team: Phoenix-content
+
+    """
```

Comment:
```suggestion
       :Steps:
       1. Generate cert files with custom_cert_generate,
         collect cacert path.
       2. Create new org, product, and http-proxy with path to cacert.
       3. Create repo using selected http proxy, perform repo sync.

    :expectedresults: http-proxy with cacert is assigned to repo,
    sync operation uses assigned http-proxy with the cert.

    :id: a9645b7f-228e-4f4d-ab04-610382bd2d0b

    :BZ: 2144044

    :customerscenario: true
    """
```
Don't need the team here since it's already set for the whole file. Also missing the leading colon before steps and I think we normally just do numbers with steps instead of bullets.